src/client/containers/SelectionPage/SelectionPage.component.js

```diff
@@ -1,5 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Selectionpage.css';
+import PropTypes from 'prop-types';
+import Radio from '../../components/Radio/ButtonRadio';
+import HyfBigLogo from '../../components/HyfBigLogo/HyfBigLogo';
+import { Button } from '../../components/Button/Button';
 
-export default function SelectionPage() {
-  return <h1>This is a SelectionPage</h1>;
-}
+const SelectionPage = (props) => {
+  const { onClick } = props;
+  const submit = () => {
+    onClick();
```

Comment:
No need to call props here - containers like page do everything by themselves. When this PR is merged, we will come back into this function and actually do something.